src/main/java/net/imglib2/algorithm/morphology/StructuringElements.java

```diff
@@ -291,6 +293,7 @@ else if ( decomposition == 8 || decomposition == 4 || decomposition == 6 )
 	 * dimensionality and target dimensionality do not match. Non-decomposed
 	 * version are dimension-generic.
 	 *
+	 * @implNote op name='create.square', type=Function
```

Comment:
`engine.create` alias?

Reply:
I'm not a huge fan of this. Most of these shapes are created by, and return, the same types, yielding a lot of conflicting results.

Honestly, let's maybe just de-opify these methods for now, as they are not used...

Reply:
On second thought, I actually think that these Ops would be pretty helpful, **except** for the fact that they return a `List<Shape>`, **not** the `Shape` itself. We should probably expose the mechanisms to just create a plain `Shape`...